rllib/examples/autoregressive_action_dist.py

```diff
@@ -10,187 +10,56 @@
 This examples shows both.
 """
 
-from gym.spaces import Discrete, Tuple
 import argparse
 
 import ray
 from ray import tune
 from ray.rllib.examples.env.correlated_actions_env import CorrelatedActionsEnv
+from ray.rllib.examples.models.autoregressive_action_model import \
```

Comment:
All Model/Distribution/Policy example classes were moved to new `examples/models`/`examples/policy` folders., so these can be found more easily and shared with other examples/test scripts.